### ISSUE TITLE
why do we use two padding tokens ('$', and '#')

### ISSUE THREAD

**Issue by yanfjz** [chenjun2hao/SRN.pytorch#7]
Thanks a lot for sharing the awesome work :)

Here I'm a little confused about the padding tokens, from the paper, it seems the author uses only one <EOS> token to pad the sentence to the batch_max_length, and I also experimented it with your implementation, however, it turns out that your implementation with two tokens produces much better result.

So I'm wondering if there is any specific reason why we chose the two-token method? Any information will be greatly appreciated.

**Comment by miaomi1994** [comment]
> Thanks a lot for sharing the awesome work :)
> 
> Here I'm a little confused about the padding tokens, from the paper, it seems the author uses only one token to pad the sentence to the batch_max_length, and I also experimented it with your implementation, however, it turns out that your implementation with two tokens produces much better result.
> 
> So I'm wondering if there is any specific reason why we chose the two-token method? Any information will be greatly appreciated.

Do you reach the paper's accuracy? I experimented and modified it as paper in some detail，however still can not get a good result,especially on the CUBE80 dataset.

**Comment by yanfjz** [comment]
@miaomi1994 nope, I was first doing quick test using my own dataset.

**Comment by mengxiaolu** [comment]
Hello, is the data format of training and testing model in MDB or other forms? Thank you very much for your reply.

**Comment by yanfjz** [comment]
> Hello, is the data format of training and testing model in MDB or other forms? Thank you very much for your reply.

Based on dataset.py in this repo, the implementation uses lmdb dataset, you can also refer to [https://github.com/clovaai/deep-text-recognition-benchmark/blob/master/create_lmdb_dataset.py](url) script for creating your lmdb dataset (the author of this SRN repo already gave the reference in the README page)
